static/app/components/events/interfaces/crashHeader/crashActions.tsx

```diff
@@ -72,15 +74,24 @@ const CrashActions = ({
   return (
     <ButtonGroupWrapper>
       <ButtonBar active={stackView} merged>
-        {hasSystemFrames && (
-          <Button
-            barId={STACK_VIEW.APP}
-            size="xsmall"
-            onClick={setStackView(STACK_VIEW.APP)}
-          >
-            {t('App Only')}
-          </Button>
-        )}
+        {hasSystemFrames &&
+          (hasGroupingTreeUI ? (
```

Comment:
can't this conditional be just around the `t(..)`?

`hasGroupingTreeUI ? t('Relevant Only') : t('App Only')`